libs/ui/src/components/MediaItem/type/VideoMedia.vue

```diff
@@ -9,7 +9,7 @@
       loop
       :autoplay="autoPlay"
       :muted="preview"
-      :poster="src"
+      :poster="placeholder"
```

Comment:
`poster` could be a meaningful image from the NFT metadata

> A URL for an image to be shown while the video is downloading. If this attribute isn't specified, nothing is displayed until the first frame is available, then the first frame is shown as the poster frame.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video#poster